game/static/game/js/tab.js

```diff
@@ -38,18 +38,23 @@ identified as the original program.
 var ocargo = ocargo || {};
 
 ocargo.Tab = function(radioElement, labelElement, paneElement) {
+    var _currentStateId;
```

Comment:
I decided to move logic related to different states allowed for a tab here.
This allowed me to remove duplication from game.js and will allow us to use CrowdIn live mode for translations (we were comparing labels see removed function _readyToPlay, _isRunning, and _isPaused in game.js).